src/theme/foundations/textStyles.ts

```diff
@@ -2,7 +2,7 @@ export const textStyles = {
   h1: {
     fontFamily: 'heading',
     fontWeight: 700,
-    fontSize: '2.75rem',
+    fontSize: { base: '2.175rem', md: '2.875rem' },
```

Comment:
I'm confused here... 
- the heading1 being removed below uses 2.125rem desktop/1.875 mobile
- this is proposing 2.875 desktop/2.175 mobile
- and the design system shows 2.175rem for both desktop and mobile

@nloureiro Could you help clarify here what the fontSize should be for the standard H1?

Reply:
Ah, never mind, I see #114 now. @nloureiro Would you mind updating the design system to reflect this?